src/routes/Send.tsx

```diff
@@ -207,6 +207,15 @@ export default function Send() {
     setFieldDestination("");
   }
 
+  const insufficientFunds = () => {
+    if (source() === "onchain") {
+      return (state.balance?.confirmed ?? 0n) < amountSats()
```

Comment:
should be the sum of confirmed + unconfirmed (I recently learned you can send from unconfirmed, changed my life)